### PR TITLE
Avoid stringify/parse from native library for get/update of elements and models

### PR DESCRIPTION
**Pull request by kabentley**

### CODE REVIEW

**Review by EarlinLutz — APPROVED** [review]

File: common/api/imodeljs-backend.api.md [hunk 1]
```diff
@@ -425,8 +428,8 @@ export class BriefcaseDb extends IModelDb {
     readonly onBeforeClose: BeEvent<() => void>;
     readonly onChangesetApplied: BeEvent<() => void>;
     static readonly onCreated: BeEvent<(_imodelDb: BriefcaseDb) => void>;
-    static readonly onOpen: BeEvent<(_requestContext: AuthorizedClientRequestContext | ClientRequestContext, _briefcaseProps: BriefcaseProps) => void>;
-    static readonly onOpened: BeEvent<(_requestContext: AuthorizedClientRequestContext | ClientRequestContext, _imodelDb: BriefcaseDb) => void>;
+    static readonly onOpen: BeEvent<(_requestContext: ClientRequestContext | AuthorizedClientRequestContext, _briefcaseProps: BriefcaseProps) => void>;
```
Comment (pmconne) [inline common/api/imodeljs-backend.api.md:431]: Bug in api-extractor causes order of these types to be reversed. Revert or extract-api will fail.

File: common/api/geometry-core.api.md [hunk 2]
```diff
@@ -3188,7 +3188,7 @@ export class Matrix3d implements BeJSONFunctions {
 }
 
 // @public
-export type Matrix3dProps = number[][] | Matrix3d | number[];
+export type Matrix3dProps = number[][] | number[];
```
Comment (pmconne) [inline common/api/geometry-core.api.md:3191]: Do you still intend to document the breaking change here and in `Texture`?

File: common/api/presentation-components.api.md [hunk 3]
```diff
@@ -165,10 +165,10 @@ export interface DataProvidersFactoryProps {
 export const DEFAULT_PROPERTY_GRID_RULESET: Ruleset;
 
 // @beta @deprecated
-export function DEPRECATED_controlledTreeWithFilteringSupport<P extends ControlledTreeWithVisibleNodesProps>(TreeComponent: React.FC<P>): React.FC<Pick<P & ControlledTreeWithFilteringSupportProps, "filter" | "onFilterApplied" | "onMatchesCounted" | "activeMatchIndex" | "nodeLoader" | "onNodeLoaderChanged" | Exclude<keyof P, "visibleNodes">>>;
+export function DEPRECATED_controlledTreeWithFilteringSupport<P extends ControlledTreeWithVisibleNodesProps>(TreeComponent: React.FC<P>): React.FC<Pick<P & ControlledTreeWithFilteringSupportProps, "filter" | "nodeLoader" | "onFilterApplied" | "onMatchesCounted" | "onNodeLoaderChanged" | "activeMatchIndex" | Exclude<keyof P, "visibleNodes">>>;
```
Comment (pmconne) [inline common/api/presentation-components.api.md:168]: probably want to `git checkout common/api/presentation-components.api.md` - another api-extractor bug.

File: common/api/imodeljs-backend.api.md (hunk 1 above)
Comment (grigasp) [reply common/api/imodeljs-backend.api.md:431]: Usually `rush clean` + `rush compile` + `rush extract-api` helps to resolve all of these at once if you don't want to revert them individually.

**Comment by pmconne** [comment]
Awaiting investigation from @grigasp re 2 failing ui-framework ModelsTree tests.

**Review by pmconne — APPROVED** [review]

**Comment by pmconne** [comment]
> Awaiting investigation from @grigasp re 2 failing ui-framework ModelsTree tests.

Fix to native code in progress - should not block this PR.

**Review by grigasp — APPROVED** [review]

**Review by DanEastBentley — APPROVED** [review]

**Review by mgooding — APPROVED** [review]

**Review by bsteinbk — APPROVED** [review]

**Pull request merged**